qutebrowser/browser/shared.py

```diff
@@ -243,3 +246,29 @@ def get_user_stylesheet():
         css += '\nhtml > ::-webkit-scrollbar { width: 0px; height: 0px; }'
 
     return css
+
+
+def netrc_authentication(url, authenticator):
```

Comment:
This removes the check we had below for whether we did already try netrc for this authentication. That's problematic, because it means we'll probably run into an endless loop in case the credentials in the netrc file is wrong...

Not sure how we can do that tracking with QtWebEngine though - setting an attribute on the QAuthenticator probably won't work? Maybe we should just keep a set of URLs where we tried to use netrc before?